content/pages/disability-benefits/claims-appeal/request-a-priority-review.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page-breadcrumbs.html
+title: Request a Priority Review
+display_title: Request a Priority Review
+order: 1
+plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
+template: detail-page
+---
+<div itemscope itemtype ="http://schema.org/HowTo">
+
+<div class="va-introtext" itemprop="description">
+If you need to have your appeal moved to the front of the line due to serious illness, financial distress, or other reasons, find out how to request a priority review.
+</div>
+
+<h3>Is there a way to prioritize my appeal?</h3>
+Advanced on Docket appeals are prioritized so that they are always at the front of the line. You can apply to have your appeal Advanced on Docket for any of the below reasons.
+
+<div class="feature">
+
+**One of these must be true. You:**
+- Are 75 years or older, **or**
+- Suffer from a serious illness, **or**
+- Are in financial distress, **or**
+- Have other sufficient cause (a reason for needing your appeal reviewed faster)
+
+**Note:** If you are older than 75, your appeal will receive Advanced on Docket status automatically.
+</div>
+
+### How do I request to have my appeal Advanced on Docket?
+You or your representative will need to write a letter to the Board of Veterans’ Appeals.
+
+**You’ll need to include all of this information:**
+- Your name (as the Veteran or other person appealing a claim—like a survivor, guardian, or fiduciary), **and**
+- The VA file number for your claim, **and**
+- The reason you want to be moved forward on the docket
+
+**You or your representative can send your written request in 1 of 2 ways:**
+
+##### By mail
+Send to: Board of Veterans’ Appeals PO Box 27063 Washington, DC 20038
+
+##### By fax
+Fax to: <a href="tel:+12024956803">(202) 495-6803</a>
```

Comment:
We occasionally include the country code in fax numbers (e.g. `pension-management-center.md`), should we do so here?